client/src/components/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./ProfilePicture.css";
+import { Link } from "react-router-dom";
+
+// Function based React Component
+const ProfilePicture = props => {
+  console.log(props);
+  // Default Class to apply to Component
+  let classList = `ProfilePicture`;
+  let types = ["extra-small", "small", "medium", "large"];
+  if (types.includes(props.type)) {
+    classList += ` profile-pic-${props.type}`;
+  }
+
+  if (props.link) {
```

Comment:
Instead of `link`, could we call the prop `to`, to be consistent with React Router?